AppCenterCrashes/AppCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -651,19 +654,161 @@ - (void)testTrackModelException {
         exception = log.exception;
       });
   [MSAppCenter configureWithAppSecret:kMSTestAppSecret];
-  [self.sut startWithLogManager:logManagerMock appSecret:kMSTestAppSecret];
```

Comment:
Any reason of this change?

Reply:
@jaelim-ms In order to be consistent with other non-public APIs (like [`trackPage` of Analytics](https://github.com/Microsoft/AppCenter-SDK-Apple/blob/develop/AppCenterAnalytics/AppCenterAnalytics/Internals/MSAnalyticsInternal.h#L24)), I changed method signature of `- (void)trackModelException:(MSException *)exception;` to static `+ (void)trackModelException:(MSException *)exception;` in `MSCrashesInternal.h`. 

As a consequence, in unit test we have to change this line to `[[MSCrashes sharedInstance] startWithLogManager:logManagerMock appSecret:kMSTestAppSecret];`.